pymongo/pool.py

```diff
@@ -1058,6 +1066,8 @@ class _PoolClosedError(PyMongoError):
     pass
 
 
+PAUSED, READY, CLOSED = range(3)
```

Comment:
Can we use a `class` enum for this like we do elsewhere (e.g. `UuidRepresentation`).

Reply:
Done.